### PR TITLE
Project Re-Structure

### PR DESCRIPTION
**Pull request by birumaka**

### CODE REVIEW

File: src/main/tests/BaseTest.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+import main.pages.LoginPageObject;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Optional;
+import org.testng.annotations.Parameters;
+
```
Comment (nimesh-mishra) [inline src/main/tests/BaseTest.java:11]: Remove the extra space.

File: src/main/tests/BaseTest.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+import java.util.concurrent.TimeUnit;
+
+public class BaseTest {
+   WebDriver driver;
+
+   @BeforeClass
+   @Parameters({"browser", "baseUrl"})
+   void setUp(@Optional("chrome") String browser, String baseUrl){
+       if (browser.equalsIgnoreCase("chrome")){
```
Comment (nimesh-mishra) [inline src/main/tests/BaseTest.java:20]:
```suggestion
       if (Constant.Chrome.equalsIgnoreCase( browser )
```
Use Constants file to maintain constants.
Create a separate file for URL constants.

File: src/main/tests/BaseTest.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+       driver.get(baseUrl);
+
+       LoginPageObject loginPageObj = new LoginPageObject(driver);
+       loginPageObj.clickLogin();
+       loginPageObj.loginUser("birushrestha@lftechnology.com", "");
+   }
+
+   /**
+   void setUp(){
```
Comment (nimesh-mishra) [inline src/main/tests/BaseTest.java:41]: Remove commented out code.

File: src/main/tests/BaseTest.java [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+           driver = new ChromeDriver();
+       }
+       driver.manage().window().maximize();
+       driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+
+       driver.get(baseUrl);
+
+       LoginPageObject loginPageObj = new LoginPageObject(driver);
+       loginPageObj.clickLogin();
+       loginPageObj.loginUser("birushrestha@lftechnology.com", "");
```
Comment (nimesh-mishra) [inline src/main/tests/BaseTest.java:37]: You can create a method in LoginPageObject to accept loginUser and click Login.

File: src/main/tests/CompanyLogoTest.java [hunk 5]
```diff
@@ -1,10 +1,13 @@
-package tests.sanity;
+package main.tests;
+/**
+ * this is test class
```
Comment (nimesh-mishra) [inline src/main/tests/CompanyLogoTest.java:3]: Explain what this test class intends to test.

File: src/main/tests/EmployeeProfileTest.java [hunk 6]
```diff
@@ -1,13 +1,16 @@
...
 import org.testng.annotations.Test;
-import pages.empProfilePageObject;
 
-public class empProfileTestBase extends sanityTestBase {
+public class EmployeeProfileTest extends BaseTest {
 
     @Test
     void getEmpDetail(){
-        empProfilePageObject empProfilePageObj = new empProfilePageObject(driver);
+        EmployeeProfilePageObject empProfilePageObj = new EmployeeProfilePageObject(driver);
```
Comment (nimesh-mishra) [inline src/main/tests/EmployeeProfileTest.java:13]: This class isn't testing anything. Remove logs afterwards.

File: src/main/tests/SearchEmployeeTimesheetDetailTest.java [hunk 7]
```diff
@@ -1,21 +1,21 @@
...
 import org.openqa.selenium.support.ui.Select;
-import org.testng.annotations.*;
-import pages.timesheetPageObject;
-
+import org.testng.annotations.Test;
 
 import static java.lang.Thread.sleep;
 
-public class searchEmpTimesheetDetailTest extends BaseTest {
+public class SearchEmployeeTimesheetDetailTest extends BaseTest {
```
Comment (nimesh-mishra) [inline src/main/tests/SearchEmployeeTimesheetDetailTest.java:14]:
```suggestion
public class SearchEmployeeTimeSheetDetailTest extends BaseTest{
```

File: src/main/tests/SearchEmployeeTimesheetSummaryTest.java [hunk 8]
```diff
@@ -1,22 +1,22 @@
...
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.ui.Select;
 import org.testng.annotations.Test;
-import pages.timesheetPageObject;
 
 import static java.lang.Thread.sleep;
 
-public class searchEmpTimesheetSummaryTest extends BaseTest {
+public class SearchEmployeeTimesheetSummaryTest extends BaseTest {
```
Comment (nimesh-mishra) [inline src/main/tests/SearchEmployeeTimesheetSummaryTest.java:15]: CamelCase class naming..

File: test execution/Second.xml [hunk 9]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
+<suite name="All Test Suite">
+    <test name="Vyaguta Second Test">
+        <parameter name="browser" value="chrome"/>
+        <parameter name="baseUrl" value="url_name"/>
+        <classes>
+            <class name="main.tests.CompanyLogoTest"/>
+            <class name="main.tests.ViewLeaveWFHTest"/>
+<!--            <class name="tests.empProfileTest"/>-->
```
Comment (nimesh-mishra) [inline test execution/Second.xml:10]: Remove these commented lines if not required.

File: src/main/tests/BaseTest.java (hunk 2 above)
Comment (birumaka) [reply src/main/tests/BaseTest.java:20]: Will be listing all the constants such as url, username, email, password and other variable that could be needed for the test in config or separate file

File: src/main/tests/BaseTest.java (hunk 4 above)
Comment (birumaka) [reply src/main/tests/BaseTest.java:37]: We do have method for both of these

**Pull request merged**